styleguide.css

```diff
@@ -364,22 +419,135 @@ p a {
 
 p a:after {
     position: absolute;
-    top:1rem;
+    top: 1rem;
     bottom: 0.5rem;
     right: 0;
     left: 0;
     display: block;
     background-color: #bde513;
     overflow: hidden;
-    content: "";
+    content: '';
     transform: translateY(35%);
     transition: all .3s;
-    z-index:-1;
+    z-index: -1;
 }
 
 p a:hover:after {
-    top:0;
+    top: 0;
     bottom: 0.5rem;
     transform: translateY(0);
     transition: all .3s;
 }
+
+/* Menu */
+
+button.menu_button {
+    position: fixed;
+    display: block;
+    top: 1rem;
+    left: 1rem;
+    width: 9.5rem;
+    height: 9.5rem;
+    border: none;
+    background: transparent;
+    padding: 0;
+    margin: 0;
+    cursor: pointer;
+    z-index: 2;
+}
+
+button.menu_button:focus {
+    outline: none;
+}
+
+button.menu_button:before,
+button.menu_button:after {
+    content: '';
+    height: 0.3rem;
+    width: 3.5rem;
+    margin: 0;
+    left: 3rem;
+    background: #00283C;
+    position: absolute;
```

Comment:
I wonder if we want a position absolute or fixed here.
How does it react if we have scrolled the page? Should we still have the button to access the menu?

Reply:
absolute is enough cause the parent is already fixed